diff --git a/src/gyp/common.gypi b/src/gyp/common.gypi
index f7257a2d7..d1d54b8bf 100644
--- a/src/gyp/common.gypi
+++ b/src/gyp/common.gypi
@@ -171,7 +171,6 @@
               '-Wtype-limits',
             ],
             'cflags_cc': [
-              '-stdlib=libc++',
               '-Wno-covered-switch-default',
               '-Wno-unnamed-type-template-args',
               '-Wno-c++11-narrowing',
@@ -191,7 +190,6 @@
               '-Wtype-limits',
             ],
             'cflags_cc': [
-              '-stdlib=libc++',
               '-Wno-covered-switch-default',
               '-Wno-unnamed-type-template-args',
               '-Wno-c++11-narrowing',
